Gemfile

```diff
@@ -35,7 +35,7 @@ gem 'uglifier', '>= 1.0.3'
 # Stanford/Hydra related gems
 gem 'about_page'
 gem 'active-fedora'
-gem 'blacklight', '~> 5.11.3' # More deprecation warnings from 5.11.3 to 5.16
+gem 'blacklight', '~> 5' # More deprecation warnings from 5.11.3 to 5.16
```

Comment:
Shouldn't we pin to `~> 5.18`? Or are we ready to go to 6.x? (and, is this comment obsolete?)